src/middlewares/validateArticle.js

```diff
@@ -18,6 +23,18 @@ const checkInput = (request, response, next) => {
       message: 'Please enter characters  between 5 and 100'
     });
   }
+  if (tags === undefined) {
+    return response.status(400).json({
+      status: 'Fail',
+      message: 'Please add a tags field',
+    });
+  }
+  if (!Array.isArray(tags)) {
+    return response.status(400).json({
+      status: 'Fail',
+      message: 'Please tags should be an array of tags',
+    });
+  }
```

Comment:
I think you can go further to validate that the tags are strings. What happens when I pass in tags that is an array of arrays

Reply:
@marcdomain, I didn't think of that. I'll think about it holistically and implement that. Thanks for the feedback.

Reply:
I've implemented that now @marcdomain, thanks for the feedback